fastplotlib/layouts/_engine.py

```diff
@@ -7,7 +7,7 @@
 from ._rect import RectManager
 
 
-class UnderlayCamera(pygfx.Camera):
+class ScreenSpaceCamera(pygfx.Camera):
     """
     Same as pygfx.ScreenCoordsCamera but y-axis is inverted.
```

Comment:
@almarklein would be useful to have a pygfx Camera that is in screenspace with inverted y, pointer events use (0, 0) as the top left corner so it would be useful to have the option.

Reply:
Something like this? https://github.com/pygfx/pygfx/pull/1092